test/functional/p2p_blocksonly.py

```diff
@@ -89,6 +89,11 @@ def blocks_relay_conn_tests(self):
         assert_equal(self.nodes[0].getpeerinfo()[0]['relaytxes'], False)
         _, txid, _, tx_hex = self.check_p2p_tx_violation()
 
+        self.log.info("Tests with node in normal mode with block-relay-only connections with relay permission")
```

Comment:
"with relay permission": 
I think that even if block-relay-only connections could have relay permission, they still wouldn't have that here because the last restart disabling `-blocksonly` mode cleared the `-whitelist=relay@127.0.0.1` arg from above.

Reply:
```suggestion
        self.log.info("Tests with node in normal mode with block-relay-only connection, sending an inv")
```

Oh good catch. I changed the test to an `inv` once I realized that permission flags can't be set, but forgot to update the comment.